src/api/api-cart/api-add-cart.ts

```diff
@@ -9,12 +9,12 @@ export interface ProductObjectInCartModel {
 
 export const apiAddCart = async (
   url: string,
-  product: ProductObjectInCartModel
+  cartObjProductUpdate: ProductObjectInCartModel
```

Comment:
Find better name. Is this really cart object but what is this ProductUpdate? Isn't it just productsCart? The type name is confusing.

Reply:
resolve